src/collective/js/cycle2/controlpanel.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+from collective.js.cycle2 import _
+from plone.app.registry.browser import controlpanel
+from plone.directives import form
+from zope import schema
+from zope.schema.vocabulary import SimpleTerm
+from zope.schema.vocabulary import SimpleVocabulary
+
+fx = ['fade', 'fadeout', 'none', 'scrollHorz']
+fx = SimpleVocabulary([SimpleTerm(f) for f in fx])
+
+# TODO: get this from portal_properties/imaging_properties
+scales = ['large', 'preview', 'mini', 'thumb', 'tile', 'icon', 'listing']
+scales = SimpleVocabulary([SimpleTerm(s) for s in scales])
```

Comment:
we have to add it to the core; plone.app.vocabularies is the pace to do it, IMO.

see: https://github.com/plone/plone.app.vocabularies/issues/11